cmd/sponged/handlers/item.go

```diff
@@ -101,7 +101,8 @@ func (itemHandle) Upsert(c *app.Context) error {
 		return err
 	}
 
-	c.Respond(nil, http.StatusNoContent)
+	c.Respond(itMap, http.StatusOK)
```

Comment:
Disemvoweling is the best way to change letters while maintaining maximum readability.